pagopa/io-app
# fix(IT Wallet): [SIW-1934] Cleanup the integrityKeyTag if getWalletAttestation throws an error

## Short description
This PR updates the `WalletInstanceAttestationObtainment` machine step adding a cleanup function if there is an error.

## List of changes proposed in this pull request
- Added cleanupIntegrityKeyTag if `getWalletAttestation` throws an error

## How to test
Ensure that, when creating a WI, after an error on the `WalletInstanceAttestationObtainment` step, the `IntegrityKeyTag` has been correctly deleted, and a new key is generated when retrying to create a WI.
